<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bdUNMainWindow</class>
 <widget class="QMainWindow" name="bdUNMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Utility Nodes</string>
  </property>
  <widget class="QWidget" name="unMainWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="masksGrp">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>180</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="2" column="1">
            <widget class="QLineEdit" name="unSrc">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Utility Node</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="unDest"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Utility Node Name</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="unPickSrcBtn">
             <property name="text">
              <string>Pick Source</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="unPickDestBtn">
             <property name="text">
              <string>Pick Destination</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QComboBox" name="unSrcAttr">
             <item>
              <property name="text">
               <string>translate</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>rotate</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>scale</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QComboBox" name="unSrcAxes">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>35</width>
               <height>20</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>X</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Y</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Z</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>All</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="1" colspan="3">
            <widget class="QComboBox" name="unComboBox"/>
           </item>
           <item row="3" column="2">
            <widget class="QComboBox" name="unDestAttr">
             <item>
              <property name="text">
               <string>translate</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>rotate</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>scale</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QComboBox" name="unDestAxes">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>20</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>X</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Y</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Z</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>All</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="4" column="1" colspan="3">
            <widget class="QLineEdit" name="unName"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="createUNBtn">
           <property name="text">
            <string>Create utility node</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="createConectionBtn">
           <property name="text">
            <string>Create Connection</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>300</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Type Filter</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="filterComboBox"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Name Filter</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="filterName"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0" rowspan="2">
          <widget class="QListWidget" name="unListView"/>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>-&gt;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>&lt;-</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" rowspan="2">
          <widget class="QListWidget" name="unConListView"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="nodeEditorBox">
      <property name="title">
       <string>Node Editor</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QFormLayout" name="nodeEditorLayout">
         <item row="0" column="1">
          <widget class="QPushButton" name="testButton">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>createUNBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
